app-modules/ai/src/Models/Prompt.php

```diff
@@ -85,13 +87,21 @@ public function upvote(): void
     {
         $this->upvotes()->create(['user_id' => auth()->id()]);
 
+        if (Feature::active('ai_utilization')) {
+            Cache::forget('prompts-liked-count');
+        }
```

Comment:
I don't think this is needed anymore since we are just having the caches be cleared and refreshed at a set interval or when a User chooses to refresh the report

Reply:
Done